examples/image_classifier/streaming_video/create_mar_file_batch.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
This file looks almost identical to the non batch version, can we refactor and delete code?

Reply:
Done